Sources/VitaminCore/Components/Badge/VitaminBadgeSize.swift

```diff
@@ -22,3 +22,16 @@ public enum VitaminBadgeSize: CaseIterable {
         }
     }
 }
+
+public extension VitaminBadgeSize {
+    var name: String {
+        switch self {
+        case .small:
+            return "Small"
+        case .medium:
+            return "Medium"
+        case .large:
+            return "Large"
+        }
+    }
+}
```

Comment:
It seems to be used only in the UIKit showcase, so please move it to the `Showcase/Application/UIKit/Components/Badge` folder.

Reply:
done